apps/src/templates/projects/ProjectsList.story.jsx

```diff
@@ -27,41 +32,26 @@ const STUB_PROJECTS_DATA = [
   }
 ];
 
-export default storybook => {
-  return storybook.storiesOf('Projects/ProjectsList', module).addStoryTable([
-    {
-      name: 'basic projects list without thumbnails column',
-      description: `This is a simple projects list with stub data.`,
-      story: () => (
-        <ProjectsList
-          projectsData={STUB_PROJECTS_DATA}
-          studioUrlPrefix={'https://studio.code.org'}
-          showProjectThumbnails={false}
-        />
-      )
-    },
-    {
-      name: 'basic projects list with thumbnails column',
-      description: `This is a simple projects list with stub data.`,
-      story: () => (
-        <ProjectsList
-          projectsData={STUB_PROJECTS_DATA}
-          studioUrlPrefix={'https://studio.code.org'}
-          showProjectThumbnails={true}
-        />
-      )
-    },
-    {
-      name: 'basic localized projects list',
-      description: `This is a simple projects list with stub data, rendered in a non-English locale`,
-      story: () => (
-        <ProjectsList
-          locale={'es-MX'}
-          projectsData={STUB_PROJECTS_DATA}
-          studioUrlPrefix={'https://studio.code.org'}
-          showProjectThumbnails={true}
-        />
-      )
-    }
-  ]);
+const DEFAULT_ARGS = {
+  projectsData: STUB_PROJECTS_DATA,
+  studioUrlPrefix: 'https://studio.code.org',
+  showProjectThumbnails: true
+};
+
+const Template = args => <ProjectsList {...args} />;
+
+// Name of export determines what is shown in storybook entry
```

Comment:
Intentionally left in?

Reply:
Nope, but with the refactor I now need it. So, now, yes!

Reply:
oh nvm, just saw in [this PR](https://github.com/code-dot-org/code-dot-org/pull/49124) a similar thing without the linting exception. Turns out my editor was complaining but it committed without issue. Removed!